metrics/bigfiles_high_water_mark

```diff
@@ -1 +1 @@
-968
+983
```

Comment:
session.py has made the list and is bumping this up. While I do expect some refactoring of this to happen in future PRs (in particular moving some of the scratch bucket and default cred functionality into the Creds class hierarchy), I'm open to ideas on how to reorganize!